app/src/main/java/com/grappus/kavach/presentation/read/ReadScreen.kt

```diff
@@ -0,0 +1,303 @@
+            identifier = "BookmarkFab"
+        ), MiniFabItem(
+            icon = ImageBitmap.imageResource(id = R.drawable.read_fab_share),
+            label = "Share",
+            identifier = "ShareFab"
+        )
+    )
+
+    val scrollState = rememberScrollState()
+
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color(0xFF0b0a07)
+
+            ),
+        contentAlignment = Alignment.Center,
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.read_bg),
+            modifier = Modifier
+                .fillMaxSize()
+                .offset { IntOffset(0, -scrollState.value) },
+            contentDescription = "Background Image",
+            contentScale = ContentScale.Crop
+        )
+
+
+    }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(state = scrollState),
+        verticalArrangement = Arrangement.Top
+    ) {
+        FloatingActionButton(onClick = { navController.popBackStack() },
+            elevation = FloatingActionButtonDefaults.elevation(0.dp),
+            modifier = Modifier.padding(top = 5.dp, start = 20.dp),
+            containerColor = Color.White.copy(alpha = 0.4f),
+            contentColor = Color.White,
+            shape = CircleShape,
+            content = {
+                Icon(
+                    imageVector = Icons.Default.KeyboardArrowLeft, contentDescription = "Back Arrow"
+                )
+            })
+        Text(
+            modifier = Modifier
+                .padding(top = 400.dp, start = 20.dp, end = 50.dp)
+                .padding(end = 50.dp),
+            text = "These are the 10 ways to rejection proof yourself!",
+            style = TextStyle(
+                fontFamily = FontFamily(Font(R.font.inter_bold)),
+                fontWeight = FontWeight(400),
+                fontSize = 28.sp,
+                lineHeight = 34.sp,
+                color = Color.White
+            )
+        )
+        Text(
+            modifier = Modifier.padding(top = 13.dp, start = 20.dp, end = 40.dp),
+            text = "Gear up to be Rejection-Proof! Check out the super cool article \"Crush Rejection with These 10 Awesome Tips!\" Learn how to bounce back like a champ, embrace feedback, and stay positive. With these epic techniques, you'll become a rejection-fighting superstar, ready to conquer any challenge! Let's rock it!",
+            style = TextStyle(
+                fontFamily = FontFamily(Font(R.font.inter_regular)),
+                fontWeight = FontWeight(400),
+                fontSize = 18.sp,
+                lineHeight = 30.sp,
+                letterSpacing = 0.1.sp,
+                color = Color.White.copy(alpha = 0.5f)
+            )
+        )
+
+        AnimatingFAB(items = fabItems)
+
+        Text(
+            "Scroll to deep dive",
+            style = TextStyle(
+                fontFamily = FontFamily(Font(R.font.inter_regular)),
```

Comment:
@HarshitGrappus font family already defined h type.kt me whi se use krlo no need to use R.font.inter_regular